indigo_app/templates/indigo_app/tasks/taxonomy_task_detail.html

```diff
@@ -0,0 +1,57 @@
+{% extends "indigo_app/tasks/tabbed_layout.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Projects" %}{% endblock %}
```

Comment:
should "Projects" be "Topics" here, or am I lost?

Reply:
fixed